specifyweb/frontend/js_src/package.json

```diff
@@ -88,6 +87,8 @@
     "@types/react-modal": "^3.13.1",
     "@types/react-router-dom": "^5.3.3",
     "@types/underscore": "^1.10.24",
+    "@typescript-eslint/eslint-plugin": "^5.59.1",
```

Comment:
please undo all changes to this file (with exception of removing `eslint-plugin-react` dependency - it's not needed here)
1. first, this is not applicable to this branch
2. secondly, you don't need to manually add dependencies for eslint plugins in here as that's already done though `@maxxxxxdlp/eslint-config` and `@maxxxxxdlp/eslint-config-react` dependencies